### PR TITLE
nodecontroller.go:format for and err

### PR DESCRIPTION
**Pull request by zhangxiaoyu-zidif**
**What this PR does / why we need it**:

format for that makes for-loop look simple and clean.
index i is not used.  use value directly of for...range would be better.

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by cblecker** [comment]
@k8s-bot ok to test
/release-note-none
/sig node

File: pkg/controller/node/nodecontroller.go [hunk 1]
```diff
@@ -740,14 +740,13 @@ func (nc *NodeController) handleDisruption(zoneToNodeConditions map[string][]*v1
 		// We're switching to full disruption mode
 		if allAreFullyDisrupted {
 			glog.V(0).Info("NodeController detected that all Nodes are not-Ready. Entering master disruption mode.")
-			for i := range nodes {
+			for _, node := range nodes {
```
Comment (wojtek-t) [inline pkg/controller/node/nodecontroller.go]: We don't want this change due to performance reasons, right?
@gmarek

Comment (gmarek) [inline pkg/controller/node/nodecontroller.go]: Yup. We generally prefer using `for i := ...`, as this doesn't involve copying object unnecessarily.

**Comment by gmarek** [comment]
This is against our performance-driven policy.

**Pull request closed without merging**